codex/rest/api.nim

```diff
@@ -249,7 +254,7 @@ proc initRestApi*(node: CodexNodeRef, conf: CodexConf): RestRouter =
       except AsyncStreamError:
         return RestApiResponse.error(Http500)
       finally:
-        await reader.closeWait()
+        await bodyStream.closeImpl()
```

Comment:
`closeImpl()` is an internal/private method and shouldn't be called directly, you should instead call `close()`.